src/schema/v2/me/__tests__/collectionsConnection.test.ts

```diff
@@ -32,17 +32,51 @@ describe("collectionsConnection", () => {
     }
   })
 
-  it("passes correct args to Gravity", async () => {
-    await runAuthenticatedQuery(query, context)
+  describe("passes correct args to Gravity", () => {
+    it("when `sort` arg is specified", async () => {
+      await runAuthenticatedQuery(query, context)
 
-    expect(context.collectionsLoader as jest.Mock).toHaveBeenCalledWith({
-      user_id: "user-42",
-      private: true,
-      saves: true,
-      sort: "-created_at",
-      offset: 0,
-      size: 1,
-      total_count: true,
```

Comment:
**suggestion:** Honestly, I don't think adding a whole new test case here is providing much value here.

We just want to know that this resolved sends the expected payload upstream.

I would leave the `it("passes correct args to Gravity")` test as it was, and only make two small changes: 
- add another `includesArtworkID` on L11 above
- add `artwork_id: "123-abc"` to the expected payload here.

Reply:
Done: https://github.com/artsy/metaphysics/pull/5022/commits/663ced4abcccf7358259c2d2991c9595afbb765d